src/main/java/au/com/sealink/quicktravel/client/models/barcodes/BaseTicket.java

```diff
@@ -0,0 +1,105 @@
+package au.com.sealink.quicktravel.client.models.barcodes;
+
+import au.com.sealink.quicktravel.client.models.barcodes.core.Format;
+import au.com.sealink.quicktravel.client.models.barcodes.core.Resource;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class BaseTicket implements ITicket {
+    @SerializedName("id")
+    @Expose
+    private int id;
+    @SerializedName("format")
+    @Expose
+    private Format format;
+    @SerializedName("reference")
+    @Expose
+    private String reference;
+    @SerializedName("resource")
+    @Expose
+    private Resource resource;
+    @SerializedName("reservation")
+    @Expose
+    private au.com.sealink.quicktravel.client.models.barcodes.core.Reservation reservation;
```

Comment:
should we import this so it's clean like Resource?

Reply:
done, dunno how I missed it